source/hwIo/ioThread.py

```diff
@@ -147,6 +147,17 @@ def start(self):
 		super().start()
 		self.handle = ctypes.windll.kernel32.OpenThread(winKernel.THREAD_SET_CONTEXT, False, self.ident)
 
+	if apcsWillBeStronglyReferenced:
```

Comment:
could this variable also be made private? I think it was added in the 2023.2 dev cycle.

Also could you update the PR description to reference the deprecation?
Thanks.

Reply:
This is now done.